src/v2/view-builder/ViewFactory.js

```diff
@@ -146,8 +160,8 @@ module.exports = {
       Logger.warn(`Cannot find customized View for ${formName}.`);
       return BaseView;
     }
-    const View = config[authenticatorType] || config[DEFAULT];
 
+    const View = config[authenticatorType] || config[DEFAULT];
```

Comment:
looks like those 2 lines are identical. still not sure why there is diff. 🤦

Reply:
I added a new line. Removed it.